src/Storages/MeiliSearch/MeiliSearchConnection.h

```diff
@@ -6,12 +6,17 @@
 #include <base/types.h>
 #include <curl/curl.h>
```

Comment:
Now this header not needed?

Reply:
I had already removed, in the last commit